core-bundle/src/Image/Studio/Figure.php

```diff
@@ -152,7 +153,7 @@ public function getSchemaOrgData(): array
      * Returns a key-value list of all link attributes. This excludes "href" by
      * default.
      */
-    public function getLinkAttributes(bool $includeHref = false): array
+    public function getLinkAttributes(bool $includeHref = false): HtmlAttributes
```

Comment:
That's a BC break, should be mentioned in the Upgrade.md :)

Reply:
Done in 24cf19c1941e28d16d83bb9d9d18dc51c82de999, thanks for the reminder. :slightly_smiling_face: